spotify_player/src/ui/page.rs

```diff
@@ -326,12 +328,25 @@ pub fn render_library_page(
     // - a playlists window
     // - a saved albums window
     // - a followed artists window
+
+    let playlist_window_width: u16;
+    let album_window_width: u16;
+
+    if configs.app_config.playlist_window_width + configs.app_config.album_window_width > 99 {
```

Comment:
I don't think this is needed. We should return an error if user specifies incorrect parameters. Otherwise, they won't know if they did something wrong